action.yml

```diff
@@ -155,3 +174,6 @@ runs:
     BITBUCKET_TOKEN: ${{ inputs.bitbucket_token }}
     BITBUCKET_USERNAME: ${{ inputs.bitbucket_username }}
     BITBUCKET_APP_PASSWORD: ${{ inputs.bitbucket_app_password }}
+    PRISMA_API_URL: ${{ inputs.PRISMA_API_URL }}
+    PRISMA_ACCESS_KEY: ${{ secrets.PRISMA_CLOUD_ACCESS_KEY }}
```

Comment:
I would refrain from hard-coding a requirement on `secrets` into the action and instead take required data as an input where a secret can be used if so desired. Additionally, the prisma api key and secret are ingestible already via the `api-key` input in the format of `<ACCESS_KEY>::<SECRET_KEY>`.